.github/workflows/aselo_beta.yml

```diff
@@ -69,37 +69,17 @@ jobs:
         with:
           ssm_parameter: "ASELO_BETA_SERVERLESS_BASE_URL"
           env_variable_name: "SERVERLESS_BASE_URL"          
-      - name: Create environment variable file
```

Comment:
A possible idea for the future:  we could move a lot of the code in lines 24-71 into the main action if we changed the variable names in AWS to use the account ids.  Then we just have to an account id variable for each.  For example, let's say we have this in `aselo_beta.yml`:
```
      ACCOUNT_SID: ${{ secrets.ASELO_BETA_ACCOUNT_SID }}
      # Call main-action to compile, deploy and check availability
      - name: Executing main-action
        uses: ./.github/actions/main-action
        with:
          account-sid: $ACCOUNT_ID
```
And we do a similar thing for each account yml file.  Then let's say that in AWS, rather than having a secret named `PROD_TWILIO_AS_API_KEY`, we have a secret named `SYNC_SERVICE_API_KEY_{{account sid}}`.  Then in the main action, we have:
```
     - name: Set Twilio Sync API key
        uses: "marvinpinto/action-inject-ssm-secrets@latest"
        with:
          ssm_parameter: format("SYNC_SERVICE_API_KEY{0}", ${{ account-sid }})
          env_variable_name: "SYNC_SERVICE_API_KEY"
     ....
```
and so on for the other secrets.  Then all of that can go into the main action and we have less code in the workflow.

No need to do this now, and we'd have to be careful about changing names for items in the parameter store.  But an idea for later.